### ISSUE TITLE
[BUG] Custom box shadows are not recognized

### ISSUE THREAD

**Issue by sbking** [francoismassart/eslint-plugin-tailwindcss#159]
**Describe the bug**
`tailwindcss/no-custom-classname` doesn't recognize custom `boxShadow`s defined in `tailwind.config.js`

**To Reproduce**
1. Add a custom `boxShadow` to the `tailwind.config.js` theme:

```js
  theme: {
    boxShadow: {
      custom: '2px 2px 2px 1px rgba(0, 0, 0, 0.2)'
    }
  }
```

2. Try to use that shadow:

```ts
<div className="shadow-custom" />
```

3. See warning: Classname 'shadow-custom' is not a Tailwind CSS class! `eslint[tailwindcss/no-custom-classname]`

**Expected behavior**
Custom box shadows should be supported

**Environment (please complete the following information):**
 - OS: macOS
 - VSCode: 1.70.2

**Comment by sbking** [comment]
Closing - needed to reference `tailwind.config.js` with `path.join(__dirname, "tailwind.config.js")`
